CHANGELOG

```diff
@@ -0,0 +1,3 @@
+## v0.1.0
```

Comment:
Can happen in a follow up, but would be great to use the same structure / header we use for the changelogs in other projects.

Reply:
Let's discuss in a future PR. The beats structure is a bit heavyweight for a small library like this, and asciidoc is less community friendly than markdown IMHO